apps/daimo-mobile/src/sync/syncLandline.ts

```diff
@@ -0,0 +1,86 @@
+import {
```

Comment:
there is logic both on the server and on the client for how to merge landline transfers with transfer clogs.

during a sync, only new transfers since the last sync get included, the remaining transfers are stored on the client side. this means that old landline transfers need to be matched and removed client side

there is server side logic to match landline transfers with on-chain transfers so that the list of transfer clogs gets returned in a clean state on full sync